src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -53,12 +57,32 @@
             ChatColor.LIGHT_PURPLE,
             ChatColor.YELLOW);
 
+    public static final List<ChatColor> CHAT_RAINBOW = Arrays.asList(
```

Comment:
wild.... You just runned /weather rainbownick wild (extracted from server logs)

https://www.dropbox.com/s/zbjz37q9bel30d9/2017-06-08-9%20%281%29.log.gz?dl=0

go and check here.